ext/unf_ext/extconf.rb

```diff
@@ -4,6 +4,18 @@
   $LDFLAGS << ' ' << `#{CONFIG['CC']} -print-file-name=libstdc++.a`.chomp
 else
   have_library('stdc++')
+  # Do a little trickery here to enable C++ standard on Solaris 11 if found.
+  # This also forces 64bit compilation mode.
+  if (RbConfig::CONFIG['host_os'] =~ /solaris(!?2.11)/)
```

Comment:
Oh wait, does `host_os` optionally include a `!` on Solaris 2.11?  Is that what you mean by this regexp?

Reply:
that looks like some mistaken copypasta.  `(?!2.11)` would be a negative lookahead assertion but `(!?2.11)` does match either `"solaris!2.11"` or `"solaris2.11"`

i'm reasonably certain that should just be `=~ /solaris2.11/)`

paging @scotthain for confirmation tho.

Reply:
This may answer what we want to match against.
irb(main):001:0> RbConfig::CONFIG['host_os']
RbConfig::CONFIG['host_os']
=> "solaris2.11"

But I would make the regex =~ /solaris2\.11/